### PR TITLE
edit pass: grpc-batch1-jd

### PR DESCRIPTION
**Pull request by Ja-Dunn**
@shirhatti   @mairaw - Please review the copy edit to these articles, and let me know if I’ve changed the technical meaning anywhere. This is for Azure DevOps work request 1646344.

Contributes to #15889

### CODE REVIEW

**Review by mairaw — COMMENTED** [review]
I really like the changes you've made @Ja-Dunn. I'm leaving a couple of comments (some unrelated to your changes). @shirhatti it would be good if could validate that the technical meaning hasn't changed anywhere.

File: docs/architecture/grpc-for-wcf-developers/appendix.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ ms.date: 09/02/2019
...
+Windows Communication Foundation (WCF) supports distributed transactions, allowing you to perform atomic operations across multiple services. This functionality is based on the [Microsoft Distributed Transaction Coordinator](https://docs.microsoft.com/previous-versions/windows/desktop/ms684146(v=vs.85)).
 
-In the modern microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies at play, including relational databases, NoSQL data stores, and messaging systems, not to mention the mix of operating systems, programming languages and frameworks that may be used in a single environment.
+In the newer, microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies involved, including relational databases, NoSQL data stores, and messaging systems. There might also be a mix of operating systems, programming languages, and frameworks in use in a single environment.
```
Comment (Youssef1313) [inline docs/architecture/grpc-for-wcf-developers/appendix.md]: I don't feel that the comma fits here.

```suggestion
In the newer microservices landscape, this type of automated distributed transaction processing isn't possible. There are too many different technologies involved, including relational databases, NoSQL data stores, and messaging systems. There might also be a mix of operating systems, programming languages, and frameworks in use in a single environment.
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/architecture/grpc-for-wcf-developers/channel-credentials.md [hunk 2]
```diff
@@ -117,11 +117,11 @@ class Program
 }
 ```
 
-## Combining ChannelCredentials and CallCredentials
+## Combine channel and call credentials
```
Comment (Youssef1313) [inline docs/architecture/grpc-for-wcf-developers/channel-credentials.md]: I think it's better to keep them as `ChannelCredentials` and `CallCredentials` as they're class names.

```suggestion
## Combine ChannelCredentials and CallCredentials
```

File: docs/architecture/grpc-for-wcf-developers/appendix.md (hunk 1 above)
Comment (pkulikov) [reply docs/architecture/grpc-for-wcf-developers/appendix.md]: Without comma, it reads as if there is another microservices landscape, which is older, has been longer on the stage. Thus, the type of processing is possible in that older microservices one.

With comma, the "newer" refers to the "landscape", not to the "microservices landscape". And, the mentioned type of processing is not possible with microservices at all.

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Comment by Ja-Dunn** [comment]
Thanks for the feedback, everyone. Fixed everything that appeared to be for me.

File: docs/architecture/grpc-for-wcf-developers/application-performance-management.md [hunk 3]
```diff
@@ -99,48 +95,48 @@ public class StockData : Stocks.StocksBase
...
 
-The ASP.NET Core gRPC client and server libraries include explicit support for DiagnosticSource and Activity, and will create activities and apply and use header information automatically.
+The ASP.NET Core gRPC client and server libraries include explicit support for `DiagnosticSource` and `Activity`, and create activities and apply and use header information automatically.
 
 > [!NOTE]
-> All of this only happens if a *listener* is consuming the diagnostic information. If there's no listener, no diagnostics are written and no activities are created.
+> All of this happens only if a listener is consuming the diagnostic information. If there's no listener, no diagnostics are written and no activities are created.
 
-### Add your own DiagnosticSources and Activities
+### Add your own diagnostic sources and activities
```
Comment (shirhatti) [inline docs/architecture/grpc-for-wcf-developers/application-performance-management.md]: `DiagnosticSource` and `System.Diagnostics.Activity` are types. I don't think they should be lower case

File: docs/architecture/grpc-for-wcf-developers/approach.md [hunk 4]
```diff
@@ -6,20 +6,20 @@ ms.date: 09/02/2019
...
-| Security (chapter 6) | Authentication, WS-Security, message encryption | Credentials, ASP.NET Core security, TLS networking |
+| Objective | Separate business code from networking implementation. | Separate business code from interface definition and networking implementation. |
+| Define services and messages (chapters 3-4)  | Service Contract, Operation Contract, and Data Contract. | Uses proto file to declare services and messages. |
+| Language (chapters 3-5) | Contracts written in C# or Visual Basic. | Protocol Buffer language. |
+| Wire format (chapter 3) | Configurable, including SOAP/XML, Plain XML, JSON, and .NET Binary. | Protocol Buffer binary format (although it's possible to use other formats).
+| Interoperability (chapter 4) | When using SOAP over HTTP. | Official support: .NET, Java, Python, JavaScript, C/C++, Go, Rust, Ruby, Swift, Dart, PHP. Unofficial support for other languages from the community. |
+| Networking (chapter 4) | Configured at runtime. Switch between NetTCP, HTTP, and MSMQ. | HTTP/2, currently over TCP only with ASP.NET Core gRPC. |
```
[the remark this answers is not in the document]
Comment (shirhatti) [reply docs/architecture/grpc-for-wcf-developers/approach.md]: Nothing else out of the box. It's reasonable to drop the "and so on" to make it an apples-to-apples comparison with gRPC

File: docs/architecture/grpc-for-wcf-developers/call-credentials.md [hunk 5]
```diff
@@ -6,27 +6,25 @@ ms.date: 09/02/2019
 
 # Call credentials
 
-Call credentials are all based on some kind of token passed in metadata with each request.
+Call credentials are all based on a token passed in metadata with each request.
 
 ## WS-Federation
 
-ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which is not supported over HTTP/2. Users are authenticated using Active Directory Federation Services (ADFS), which provides a token that can be used to authenticate with ASP.NET Core.
+ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which isn't supported over HTTP/2. Users authenticate by using Active Directory Federation Services (AD FS), which provides a token to authenticate with ASP.NET Core.
```
Comment (shirhatti) [inline docs/architecture/grpc-for-wcf-developers/call-credentials.md]:
```suggestion
ASP.NET Core supports WS-Federation using the [WsFederation](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.WsFederation) NuGet package. WS-Federation is the closest available alternative to Windows Authentication, which isn't supported over HTTP/2. Users authenticate by using Active Directory Federation Services (AD FS), which provides a token that can be used to authenticate with ASP.NET Core.
```

I feel like this changes the meaning. AD FS doesn't directly authenticate with ASP.NET Core so it's important to make that distinction that you're only issued a token.

File: docs/architecture/grpc-for-wcf-developers/appendix.md [hunk 6]
```diff
@@ -6,17 +6,17 @@ ms.date: 09/02/2019
...
 
-If that isn't possible, then one alternative is to use the [Saga pattern](https://microservices.io/patterns/data/saga.html). In a saga, updates are processing sequentially; as each update succeeds the next one is triggered. These triggers can be propagated from service to service, or managed by a saga coordinator or "orchestrator". If an update fails at any point during the process, the services that have already completed their updates apply specific logic to reverse them.
+If that isn't possible, then one alternative is to use the [Saga pattern](https://microservices.io/patterns/data/saga.html). In a saga, updates process sequentially; as each update succeeds, the next one is triggered. These triggers can be propagated from service to service, or managed by a saga coordinator or orchestrator. If an update fails at any point during the process, the services that have already completed their updates apply specific logic to reverse them.
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/architecture/grpc-for-wcf-developers/appendix.md]:
```suggestion
If that isn't possible, then one alternative is to use the [Saga pattern](https://microservices.io/patterns/data/saga.html). In a saga, updates are processed sequentially; as each update succeeds, the next one is triggered. These triggers can be propagated from service to service, or managed by a saga coordinator or orchestrator. If an update fails at any point during the process, the services that have already completed their updates apply specific logic to reverse them.
```

File: docs/architecture/grpc-for-wcf-developers/application-performance-management.md (hunk 3 above)
Comment (mairaw) [reply docs/architecture/grpc-for-wcf-developers/application-performance-management.md]: It's weird that they would be written in plural form before if they are API names. 

So maybe

```suggestion
### Add your own `DiagnosticSource` and `Activity`
```

**Comment by mairaw** [comment]
@Ja-Dunn you can batch suggestions into a single commit using the Files changed tab.

**Comment by Ja-Dunn** [comment]
@mairaw - Thanks for the tip on batching commits! Looks like I still didn't quite get it, although I did go to the Files changed tab to try to do so. Anyway, okay, I believe I captured all the feedback.
@shirhatti - Appreciate the close reading. Re: "first class" description of support, I was originally reading that as a marketing claim, and we typically avoid that in technical docs. That said, I understood the nuance you were trying to get at in your comment. I think "built-in" is a great alternative phrasing.

**Comment by mairaw** [comment]
LGTM. I'll let @shirhatti do the approval before merging.

**Review by shirhatti — APPROVED** [review]

**Pull request merged**